docs/about/security.md

```diff
@@ -70,17 +70,17 @@ The process of releasing our source-code artifacts is highly automated yet manda
 
 ### Resilient PR and Approval Processes
 
-Our WG-TECH reviews code changes, tags PRs with scope, and makes sure that everything stays up to date. And when its time to publish a new version, one of the maintainers tags a new release on master, which:
+Our WG-TECH reviews code changes, tags PRs with scope, and makes sure that everything stays up to date. And when its time to publish a new version, one of the maintainers tags a new release on main, which:
```

Comment:
actually it's the dev branch, but idk if that's important 🤷

Reply:
The most important!